scripts/ci/vagrant.sh

```diff
@@ -6,29 +6,55 @@
 set -e
 set -x
 
-VAGRANT_VERSION=2.2.7
+VAGRANT_VERSION=2.2.10
 FEDORA_VERSION=32
```

Comment:
Should we also update to Fedora 33?

Reply:
We can do that, yes, but it is not really important to get this PR initially merged.